File: app/src/main/java/br/com/levimendesestudos/spidermagazine/mvp/contracts/BasicView.java
package br.com.levimendesestudos.spidermagazine.mvp.contracts;

/**
 * Created by 809778 on 21/08/2017.
 */

public interface BasicView {

    boolean isActive();

    void showToast(String msg);

    void showToast(int msg);

    boolean hasInternet();

}

File: app/src/main/java/br/com/levimendesestudos/spidermagazine/mvp/contracts/BasicView.kt
package br.com.levimendesestudos.spidermagazine.mvp.contracts

/**
 * Created by 809778 on 21/08/2017.
 */

interface BasicView {

    fun active(): Boolean

    fun showToast(msg: String)

    fun showToast(msg: Int)

    fun hasInternet(): Boolean

}